### PR TITLE
Create Willhe1995's HP recommandation file

### PR DESCRIPTION
**Pull request by willhe1995**
- This is a bullet point.  Willhe1995
- [ ] This is a task list item. 1

### CODE REVIEW

**Comment by pdietl** [comment]
Hello there

File: willhe1995-hollypond.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Holly Pond, Alabama
+
+##Places to Eat
+- Brandin Iron
+- Parkside Catfish
+- All Steak
+
+##Places to Visit
+- Ave Maria Grotto
+- Strawberry Festival
```
Comment (pdietl) [inline willhe1995-hollypond.md]: Perhaps I should visit

**Review by pdietl — APPROVED** [review]
Dairy Queen is bomb :bomb: 

:confetti_ball:

**Comment by willhe1995** [comment]
Merged.

**Pull request merged**